packages/pinorama-studio/src/hooks/use-pinorama-connection.ts

```diff
@@ -10,43 +10,52 @@ export type ConnectionStatus =
   | "failed"
   | "unknown"
 
+export const ConnectionStatus = Object.freeze({
+  Disconnected: "disconnected",
+  Connecting: "connecting",
+  Connected: "connected",
+  Failed: "failed",
+  Unknown: "unknown"
+}) satisfies Readonly<Record<Capitalize<ConnectionStatus>, ConnectionStatus>>
+
+type ConnectionStatusDetail = {
+  connectionStatus: ConnectionStatus
```

Comment:
To make the type more concise and avoid redundancy, we can rename the property `connectionStatus` to `status` since the type is already connection-centric. Thanks!

Reply:
Ok but that would require a wider change because all the pieces where connectionStatus is used need to change as well

Reply:
OK, we can refactor this in a separate commit.